### ISSUE TITLE
Missing statement types in OperationVisitor

### ISSUE THREAD

**Issue by bkoelman** [dotnet/roslyn#27036]
**Version Used**: 
master branch

`OperationVisitor` is missing virtual methods for `unsafe` and `checked` statements.
`fixed` statements are also missing (see https://github.com/dotnet/roslyn/issues/21281).

Please add support for these.

**Comment by jcouv** [comment]
Tagging @AlekseyTs @333fred

**Comment by AlekseyTs** [comment]
I am not sure if ```checked``` are useful, the individual operations that are affected by this context are reflecting what semantics is being used.

**Comment by bkoelman** [comment]
It would be useful for my analyzer that counts statements.

**Comment by AlekseyTs** [comment]
> It would be useful for my analyzer that counts statements.

I don't think this among the top priorities to address for IOperation nodes. If full syntactic fidelity is needed, a syntax tree should be used.
